Makefile

```diff
@@ -43,13 +43,15 @@ CC_TYPE = cc
 CC_PROJECT_NAME = Intel® Clear Containers
 CC_PROJECT_TAG = clear-containers
 CC_PROJECT_URL = https://github.com/clearcontainers
+CC_BUG_URL = $(CC_PROJECT_URL)/runtime/issues/new
 
 # build type for Kata Containers
 KATA_TYPE = kata
 
 KATA_PROJECT_NAME = Kata Containers
 KATA_PROJECT_TAG = kata-containers
 KATA_PROJECT_URL = https://github.com/kata-containers
+KATA_BUG_URL = $(KATA_PROJECT_URL)/kata-containers/issues/new
```

Comment:
I guess that is in theory the best, most generic and neutral, URL we have. Shame 'orgs' don't have an Issues section on github.

Reply:
Yeah. I suspect we'll end up with a bunch of issues both at this URL and at the runtime one but from a triage perspective, it would be good if we could get them reported in the "main" repo as defined here.